charts/pravega/README.md

```diff
@@ -99,3 +99,5 @@ The following table lists the configurable parameters of the pravega chart and t
 | `storage.cache.className` | Storage class for cache volume | `` |
 | `storage.cache.size` | Storage requests for cache volume | `20Gi` |
 | `options` | List of Pravega options | |
+| `segmentStoreSecurityContext.runAsUser` | The UID to run the entrypoint of the container process | `0` |
```

Comment:
Can we mention that by default we are not setting any value for security context.

Reply:
Done.